.changeset/wicked-dancers-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+adds center mixin
```

Comment:
```suggestion
Added a new `center` style mixin to center content horizontally and vertically.
```

Basically the changeset body is the changelog entry, so we try to make it a bit more explicit 🙂

Reply:
Thanks for the improvement:)